Jellyfin.Plugin.KodiSyncQueue/Jellyfin.Plugin.KodiSyncQueue.csproj

```diff
@@ -6,6 +6,10 @@
     <FileVersion>2.0.3.0</FileVersion>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
+    <DebugType>full</DebugType>
```

Comment:
umm, what's this?

Reply:
Just a failed attempt at making it build with deps. As I said, csproj is still a TODO ;)